src/deadline/blender_submitter/addons/deadline_cloud_blender_submitter/open_deadline_cloud_dialog.py

```diff
@@ -142,22 +146,28 @@ def _create_bundle(
         settings: The job settings.
         queue_parameters: The queue parameters.
         asset_references: The asset references.
-        requirements: The host requirements. There are only passed if the user has specified custom host requirements in the UI. If "all available worker hosts" are selected, this is None.
+        host_requirements: The host requirements. There are only passed if the user has specified custom host requirements in the UI. If "all available worker hosts" are selected, this is None.
```

Comment:
nit: grammar

This is technically out-of-scope since it has not changed, but if you're pushing a second rev, we might as well fix it up.

```suggestion
        host_requirements: The host requirements. These are only passed if the user has specified custom host requirements in the UI. If "all available worker hosts" are selected, this is None.
```